### ISSUE TITLE
Requires trailing comma in empty list

### ISSUE THREAD

**Issue by jlous** [pinterest/ktlint#2863]
## Expected Behavior
This code should pass:
```
@ContextConfiguration(
    initializers = [
    ],
) 
abstract class SpringSpec(...
```

## Observed Behavior
SringSpec.kt:15:21 Missing trailing comma before "]"

Such a comma would not be not legal Kotlin.

It seems the rule requires "a comma at the end of any (multiline) list", rather than the more sensible "a comma after each list item".

Yes, I can make it go away by putting the empty list on a single line, or commenting out the entire clause, but those are disruptive, (especially when the list is empty because items are temporarily commented out), and in any case orthogonal to what the error is complaining about.

## Your Environment
* Version of ktlint used: 0.50.0
* Gradle plugin

**Comment by paul-dingemans** [comment]
This is indeed a bug. But please do note that it will not be fixed in 0.x versions. It is really time to migrate to the 1.x releases ;-) Current release is `1.4.1`.
